src/snippets/uncurated-ncbi-dataset.rst

```diff
@@ -0,0 +1,37 @@
+1. Enter an interactive Nextstrain shell to be able to run the NCBI Datasets CLI commands without installing them separately.
+
+.. code-block::
+
+    $ nextstrain shell .
+
+2. Create the ``ingest/data`` directory if it doesn't already exist.
+
+.. code-block::
+
+    $ mkdir -p ingest/data
+
+3. Download the dataset with the pathogen NCBI taxonomy ID.
+
+.. code-block::
+
+    $ datasets download virus genome taxon <taxon-id> \
+        --filename ingest/data/ncbi_dataset.zip
+
+4. Extract and format the metadata as a TSV file for easy inspection
+
+.. code-block::
+
+    $ dataformat tsv virus-genome \
+        --package ingest/data/ncbi_dataset.zip \
+        > ingest/data/raw_metadata.tsv
+
+5. Exit the Nextstrain shell to return to your usual shell environment.
+
+.. code-block::
+
+    $ exit
+
+The produced ``ingest/data/raw_metadata.tsv`` will contain all of the fields available from NCBI Datasets.
+Note that the headers in this file use the human readable ``Name`` of the
+`NCBI Datasets' available fields <https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/dataformat/tsv/dataformat_tsv_virus-genome/#fields>`_,
```

Comment:
When designing the ingest workflow did you consider having an intermediate output which was exactly this (`raw_metadata.tsv`, or maybe 2 files if the sequences were split out into a FASTA)? This would remove the need for users to run these 5 steps which in turn would encourage comparing the NCBI data vs the curated data?

The files aren't too large, so I don't think space is a concern, but if it were we could easily mark them with `temp()` and then this section would be "run with `--notemp` to get the raw NCBI data to compare against the curated data".

Reply:
No, I had not...This is definitely left over from working with the SARS-CoV-2 data, where I had focused on only using metadata columns that are needed for the workflow to avoid large files.

Reply:
For huge datasets this is the right call, but we don't need to build every pipeline to handle the data requirements of SC2.

Reply:
I updated the [default config in the pathogen-repo-guide](https://github.com/nextstrain/pathogen-repo-guide/pull/38) to include all of the fields of the raw metadata so that people can remove fields as needed.

Reply:
Updated tutorial to use the new Snakemake target from the pathogen-repo-guide in https://github.com/nextstrain/docs.nextstrain.org/pull/195/commits/43b0f778647bd1bc1156bc25f964390bcff7e9e9